app/api/paddle/transactions/route.ts

```
@@ -0,0 +1,94 @@
+      : undefined;
+    const status = searchParams.get('status')?.split(',') || undefined;
+
+    switch (action) {
+      case 'getById':
+        if (!transactionId) {
+          return NextResponse.json(
+            { error: 'Transaction ID required' },
+            { status: 400 }
+          );
+        }
+        const transaction = await getTransactionById(transactionId);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)